### ISSUE TITLE
Getting a lot of INFO messges in the server console

### ISSUE THREAD

**Issue by hrstoyanov** [orientechnologies/orientdb#8676]
### OS: MacOs 

If I let the ODB server run for a while on my Mac, hibernate the Mac, then come back, I see a lot of these messages - why do get them and are these errors?
  
```
2018-11-25 00:20:41:271 INFO  No reason to make fuzzy checkpoint [OLocalPaginatedStorage]
2018-11-25 00:21:26:134 INFO  Before fuzzy checkpoint: min LSN segment is 3, WAL begin is OLogSequenceNumber{segment=3, position=18}, WAL end is OLogSequenceNumber{segment=3, position=107716}, fuzzy segment is 3 [OLocalPaginatedStorage]
2018-11-25 00:21:26:135 INFO  No reason to make fuzzy checkpoint [OLocalPaginatedStorage]
2018-11-25 00:25:41:264 INFO  Before fuzzy checkpoint: min LSN segment is 1, WAL begin is OLogSequenceNumber{segment=1, position=18}, WAL end is OLogSequenceNumber{segment=1, position=553700}, fuzzy segment is 1 [OLocalPaginatedStorage]
2018-11-25 00:25:41:264 INFO  No reason to make fuzzy checkpoint [OLocalPaginatedStorage]
2018-11-25 00:26:26:128 INFO  Before fuzzy checkpoint: min LSN segment is 3, WAL begin is OLogSequenceNumber{segment=3, position=18}, WAL end is OLogSequenceNumber{segment=3, position=107716}, fuzzy segment is 3 [OLocalPaginatedStorage]
2018-11-25 00:26:26:128 INFO  No reason to make fuzzy checkpoint [OLocalPaginatedStorage]
2018-11-25 00:30:41:257 INFO  Before fuzzy checkpoint: min LSN segment is 1, WAL begin is OLogSequenceNumber{segment=1, position=18}, WAL end is OLogSequenceNumber{segment=1, position=553700}, fuzzy segment is 1 [OLocalPaginatedStorage]
2018-11-25 00:30:41:258 INFO  No reason to make fuzzy checkpoint [OLocalPaginatedStorage]
2018-11-25 00:31:26:125 INFO  Before fuzzy checkpoint: min LSN segment is 3, WAL begin is OLogSequenceNumber{segment=3, position=18}, WAL end is OLogSequenceNumber{segment=3, position=107716}, fuzzy segment is 3 [OLocalPaginatedStorage]
2018-11-25 00:31:26:125 INFO  No reason to make fuzzy checkpoint [OLocalPaginatedStorage]
2018-11-25 00:35:41:235 INFO  Before fuzzy checkpoint: min LSN segment is 1, WAL begin is OLogSequenceNumber{segment=1, position=18}, WAL end is OLogSequenceNumber{segment=1, position=553700}, fuzzy segment is 1 [OLocalPaginatedStorage]
2018-11-25 00:35:41:235 INFO  No reason to make fuzzy checkpoint [OLocalPaginatedStorage]
2018-11-25 00:36:26:100 INFO  Before fuzzy checkpoint: min LSN segment is 3, WAL begin is OLogSequenceNumber{segment=3, position=18}, WAL end is OLogSequenceNumber{segment=3, position=107716}, fuzzy segment is 3 [OLocalPaginatedStorage]
2018-11-25 00:36:26:100 INFO  No reason to make fuzzy checkpoint [OLocalPaginatedStorage]
2018-11-25 00:40:41:231 INFO  Before fuzzy checkpoint: min LSN segment is 1, WAL begin is OLogSequenceNumber{segment=1, position=18}, WAL end is OLogSequenceNumber{segment=1, position=553700}, fuzzy segment is 1 [OLocalPaginatedStorage]
2018-11-25 00:40:41:232 INFO  No reason to make fuzzy checkpoint [OLocalPaginatedStorage]
2018-11-25 00:41:26:092 INFO  Before fuzzy checkpoint: min LSN segment is 3, WAL begin is OLogSequenceNumber{segment=3, position=18}, WAL end is OLogSequenceNumber{segment=3, position=107716}, fuzzy segment is 3 [OLocalPaginatedStorage]
2018-11-25 00:41:26:092 INFO  No reason to make fuzzy checkpoint [OLocalPaginatedStorage]
2018-11-25 00:45:41:227 INFO  Before fuzzy checkpoint: min LSN segment is 1, WAL begin is OLogSequenceNumber{segment=1, position=18}, WAL end is OLogSequenceNumber{segment=1, position=553700}, fuzzy segment is 1 [OLocalPaginatedStorage]
2018-11-25 00:45:41:227 INFO  No reason to make fuzzy checkpoint [OLocalPaginatedStorage]
2018-11-25 00:46:26:086 INFO  Before fuzzy checkpoint: min LSN segment is 3, WAL begin is OLogSequenceNumber{segment=3, position=18}, WAL end is OLogSequenceNumber{segment=3, position=107716}, fuzzy segment is 3 [OLocalPaginatedStorage]
2018-11-25 00:46:26:087 INFO  No reason to make fuzzy checkpoint [OLocalPaginatedStorage]
2018-11-25 00:50:41:223 INFO  Before fuzzy checkpoint: min LSN segment is 1, WAL begin is OLogSequenceNumber{segment=1, position=18}, WAL end is OLogSequenceNumber{segment=1, position=553700}, fuzzy segment is 1 [OLocalPaginatedStorage]
2018-11-25 00:50:41:223 INFO  No reason to make fuzzy checkpoint [OLocalPaginatedStorage]
2018-11-25 00:51:26:077 INFO  Before fuzzy checkpoint: min LSN segment is 3, WAL begin is OLogSequenceNumber{segment=3, position=18}, WAL end is OLogSequenceNumber{segment=3, position=107716}, fuzzy segment is 3 [OLocalPaginatedStorage]
2018-11-25 00:51:26:078 INFO  No reason to make fuzzy checkpoint [OLocalPaginatedStorage]
2018-11-25 00:55:41:217 INFO  Before fuzzy checkpoint: min LSN segment is 1, WAL begin is OLogSequenceNumber{segment=1, position=18}, WAL end is OLogSequenceNumber{segment=1, position=553700}, fuzzy segment is 1 [OLocalPaginatedStorage]
2018-11-25 00:55:41:218 INFO  No reason to make fuzzy checkpoint [OLocalPaginatedStorage]
2018-11-25 00:56:26:069 INFO  Before fuzzy checkpoint: min LSN segment is 3, WAL begin is OLogSequenceNumber{segment=3, position=18}, WAL end is OLogSequenceNumber{segment=3, position=107716}, fuzzy segment is 3 [OLocalPaginatedStorage]
2018-11-25 00:56:26:069 INFO  No reason to make fuzzy checkpoint [OLocalPaginatedStorage]
2018-11-25 05:32:06:208 INFO  Before fuzzy checkpoint: min LSN segment is 1, WAL begin is OLogSequenceNumber{segment=1, position=18}, WAL end is OLogSequenceNumber{segment=1, position=553700}, fuzzy segment is 1 [OLocalPaginatedStorage]
2018-11-25 05:32:06:210 INFO  No reason to make fuzzy checkpoint [OLocalPaginatedStorage]
2018-11-25 05:32:51:057 INFO  Before fuzzy checkpoint: min LSN segment is 3, WAL begin is OLogSequenceNumber{segment=3, position=18}, WAL end is OLogSequenceNumber{segment=3, position=111721}, fuzzy segment is 3 [OLocalPaginatedStorage]
2018-11-25 05:32:51:058 INFO  No reason to make fuzzy checkpoint [OLocalPaginatedStorage]
2018-11-25 05:37:06:208 INFO  Before fuzzy checkpoint: min LSN segment is 1, WAL begin is OLogSequenceNumber{segment=1, position=18}, WAL end is OLogSequenceNumber{segment=1, position=553700}, fuzzy segment is 1 [OLocalPaginatedStorage]
2018-11-25 05:37:06:209 INFO  No reason to make fuzzy checkpoint [OLocalPaginatedStorage]
2018-11-25 05:37:51:051 INFO  Before fuzzy checkpoint: min LSN segment is 3, WAL begin is OLogSequenceNumber{segment=3, position=18}, WAL end is OLogSequenceNumber{segment=3, position=111721}, fuzzy segment is 3 [OLocalPaginatedStorage]
2018-11-25 05:37:51:051 INFO  No reason to make fuzzy checkpoint [OLocalPaginatedStorage]
2018-11-25 05:42:06:201 INFO  Before fuzzy checkpoint: min LSN segment is 1, WAL begin is OLogSequenceNumber{segment=1, position=18}, WAL end is OLogSequenceNumber{segment=1, position=553700}, fuzzy segment is 1 [OLocalPaginatedStorage]
2018-11-25 05:42:06:201 INFO  No reason to make fuzzy checkpoint [OLocalPaginatedStorage]
2018-11-25 05:42:51:044 INFO  Before fuzzy checkpoint: min LSN segment is 3, WAL begin is OLogSequenceNumber{segment=3, position=18}, WAL end is OLogSequenceNumber{segment=3, position=111721}, fuzzy segment is 3 [OLocalPaginatedStorage]
2018-11-25 05:42:51:045 INFO  No reason to make fuzzy checkpoint [OLocalPaginatedStorage]
2018-11-25 05:47:10:120 INFO  Before fuzzy checkpoint: min LSN segment is 1, WAL begin is OLogSequenceNumber{segment=1, position=18}, WAL end is OLogSequenceNumber{segment=1, position=553700}, fuzzy segment is 1 [OLocalPaginatedStorage]
2018-11-25 05:47:10:121 INFO  No reason to make fuzzy checkpoint [OLocalPaginatedStorage]
2018-11-25 05:47:54:965 INFO  Before fuzzy checkpoint: min LSN segment is 3, WAL begin is OLogSequenceNumber{segment=3, position=18}, WAL end is OLogSequenceNumber{segment=3, position=111721}, fuzzy segment is 3 [OLocalPaginatedStorage]
2018-11-25 05:47:54:965 INFO  No reason to make fuzzy checkpoint [OLocalPaginatedStorage]
2018-11-25 05:52:10:114 INFO  Before fuzzy checkpoint: min LSN segment is 1, WAL begin is OLogSequenceNumber{segment=1, position=18}, WAL end is OLogSequenceNumber{segment=1, position=553700}, fuzzy segment is 1 [OLocalPaginatedStorage]
2018-11-25 05:52:10:114 INFO  No reason to make fuzzy checkpoint [OLocalPaginatedStorage]
2018-11-25 05:52:54:959 INFO  Before fuzzy checkpoint: min LSN segment is 3, WAL begin is OLogSequenceNumber{segment=3, position=18}, WAL end is OLogSequenceNumber{segment=3, position=111721}, fuzzy segment is 3 [OLocalPaginatedStorage]
2018-11-25 05:52:54:960 INFO  No reason to make fuzzy checkpoint [OLocalPaginatedStorage]
2018-11-25 05:57:10:109 INFO  Before fuzzy checkpoint: min LSN segment is 1, WAL begin is OLogSequenceNumber{segment=1, position=18}, WAL end is OLogSequenceNumber{segment=1, position=553700}, fuzzy segment is 1 [OLocalPaginatedStorage]
2018-11-25 05:57:10:110 INFO  No reason to make fuzzy checkpoint [OLocalPaginatedStorage]
2018-11-25 05:57:54:956 INFO  Before fuzzy checkpoint: min LSN segment is 3, WAL begin is OLogSequenceNumber{segment=3, position=18}, WAL end is OLogSequenceNumber{segment=3, position=111721}, fuzzy segment is 3 [OLocalPaginatedStorage]
2018-11-25 05:57:54:956 INFO  No reason to make fuzzy checkpoint [OLocalPaginatedStorage]
```

**Comment by laa** [comment]
Hi @hrstoyanov , no those not are errors. It is just status messages in the next release we will hide them.

**Comment by laa** [comment]
Fixed in latest 3.0.x
